### ISSUE TITLE
Memories Mod

### ISSUE THREAD

**Issue by ghost** [StardewModders/mod-ideas#211]
A mod that allows you to relive all or some of the cutscenes in the game (bachelors, bachelorettes, wedding, etc). It will be placed as a memory album inside the house.

**Comment by skuldomg** [comment]
Started fiddling around: https://github.com/skuldomg/Memories

**Comment by tnt-2018** [comment]
@skuldomg make sure to say if you would like to take on this mod idea. If you do, I or another Member could set the label as "In Progress" so no one else takes it. :)

**Comment by skuldomg** [comment]
I thought that would imply I'm taking it on :) So yeah, consider it taken on!

**Comment by tnt-2018** [comment]
Yeah, I didn't want to assume that you were taking it on, and set it as "In Progress". Just wanted to clarify. But thanks for taking on some of the communities's ideas! :)

**Comment by CoxinhaX** [comment]
This mod lets you replay memories, it's not a album but you get a awesome cave instead :D

**Comment by tiakall** [comment]
Mobile Phone Continued allows replaying events as well!  https://www.nexusmods.com/stardewvalley/mods/21017  There are likely other mods that do this too.
